src/Client/V2/Client.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Lalamove\Client\V2;
+
+use Lalamove\Resources\V2\DriversResource;
+use Lalamove\Resources\V2\OrdersResource;
+use Lalamove\Resources\V2\QuotationsResource;
+
+class Client
+{
+    protected $settings;
+
+    public function __construct(Settings $settings)
+    {
+        $this->settings = $settings;
+    }
+
+    /**
+     * @return \Lalamove\Resources\V2\OrdersResource
+     */
+    public function orders()
```

Comment:
Return types can be explicitly typed and the doc block removed. E.g:

```php
    /**
     * @return \Lalamove\Resources\V2\OrdersResource
     */
    public function orders()
    {
       // ...
    }
```

... becomes:

```php
    public function orders(): OrdersResource
    {
       // ...
    }
```